src/issueHandling/issueMapping.ts

```diff
@@ -33,6 +33,12 @@ export function addIssueToIssueList(issue: Issue) {
 	foundIssues.push(issue);
 }
 
+export function concatOneIssueList(issues: Issue[]) {
+	for (const issue of issues) {
+		addIssueToIssueList(issue);
+	}
+}
```

Comment:
This method breaks the encapsulation of the issueMapping-module. Please remove.

Reply:
What do you mean by encapsulation? I am not sure, if I have the same understanding of this word as you. In my mind it refers to the ability to manipulate the used data structures. Also I only use a method that was used before, so isn't this first method addIssueToIssueList breaking the encapsulation as well?
Also, I only use this new method on an Issue[] that I don't want to manipulate anyways.

Reply:
IMHO we shouldn't allow to add `Issue`-instances to the issue-list directly.